lib/need_for_type/states/menu_state.rb

```diff
@@ -0,0 +1,20 @@
+module NeedForType::States
+  class MenuState < State
+
+    def initialize(display_window)
+      super(display_window)
+    end
+
+    def input_worker(number_option)
+      input = @display_window.get_input
+
+      if input == Curses::Key::UP
+        @option = (@option - 1) % number_option
+      elsif input == Curses::Key::DOWN
+        @option = (@option + 1) % number_option
+      elsif input == Curses::Key::ENTER || input == 10
```

Comment:
Since we are repeating `exit if @option == <number_option - 1>` on every input_worker block, I think we can add it here, before the `yield`.